src/tools/Repl/Program.cs

```diff
@@ -0,0 +1,601 @@
+                        var column = 0;
+                        foreach (var field in row.Value.Fields)
+                        {
+                            columnWidth[column] = Math.Max(columnWidth[column], PrintResult(field.Value, true).Length);
+                            column++;
+                        }
+                    }
+
+                    // special treatment for single column table named Value
+                    if (columnWidth.Length == 1 && table.Rows.First().Value.Fields.First().Name == "Value")
+                    {
+                        var separator = string.Empty;
+                        resultString = "[";
+                        foreach (var row in table.Rows)
+                        {
+                            resultString += separator + PrintResult(row.Value.Fields.First().Value);
+                            separator = ", ";
+                        }
+
+                        resultString += "]";
+                    }
+
+                    // otherwise a full table treatment is needed
+                    else if (_formatTable)
+                    {
+                        resultString = "\n ";
+                        var column = 0;
+                        foreach (var field in table.Rows.First().Value.Fields)
+                        {
+                            columnWidth[column] = Math.Max(columnWidth[column], field.Name.Length);
+                            resultString += " " + field.Name.PadLeft(columnWidth[column]) + "  ";
+                            column++;
+                        }
+
+                        resultString += "\n ";
+
+                        foreach (var width in columnWidth)
+                        {
+                            resultString += new string('=', width + 2) + " ";
+                        }
+
+                        foreach (var row in table.Rows)
+                        {
+                            column = 0;
+                            resultString += "\n ";
+                            foreach (var field in row.Value.Fields)
+                            {
+                                resultString += " " + PrintResult(field.Value, true).PadLeft(columnWidth[column]) + "  ";
+                                column++;
+                            }
+                        }
+                    }
+                    else
+                    {
+                        // table without formatting 
+
+                        resultString = "[";
+                        var separator = string.Empty;
+                        foreach (var row in table.Rows)
+                        {
+                            resultString += separator + PrintResult(row.Value);
+                            separator = ", ";
+                        }
+
+                        resultString += "]";
+                    }
+                }
+            }
+            else
+            {
+                resultString = value.ToExpression();
```

Comment:
This is also a nice simplification - we now have more reliable ToString functions for FormulaValues.